api/v1alpha8/types.go

```diff
@@ -128,6 +128,12 @@ type PortOpts struct {
 	ValueSpecs []ValueSpec `json:"valueSpecs,omitempty"`
 }
 
+type PortStatus struct {
```

Comment:
Note for the reviewers: I think we'll have more than the `ID` field in the future, which is why I prefer to call it `PortStatus` rather than `PortID` for example.